release/start-acs/start-acs.sh

```diff
@@ -28,28 +28,81 @@ pushd "$STACKROX_DIR"
 # shellcheck source=/dev/null
 source "${STACKROX_DIR}"/deploy/common/deploy.sh
 
+gh_log notice "Deploying central..."
 "${STACKROX_DIR}"/deploy/k8s/central.sh
+
+gh_log notice "Forwarding central port..."
 kubectl -n stackrox port-forward deploy/central 8000:8443 > /dev/null 2>&1 &
 sleep 20
 
+gh_log notice "Deploying sensor..."
 "${STACKROX_DIR}"/deploy/k8s/sensor.sh
 
-kubectl -n stackrox set env deploy/sensor MUTEX_WATCHDOG_TIMEOUT_SECS=0 ROX_FAKE_KUBERNETES_WORKLOAD=long-running ROX_FAKE_WORKLOAD_STORAGE=/var/cache/stackrox/pebble.db
-kubectl -n stackrox patch deploy/sensor -p '{"spec":{"template":{"spec":{"containers":[{"name":"sensor","resources":{"requests":{"memory":"3Gi","cpu":"2"},"limits":{"memory":"12Gi","cpu":"4"}}}]}}}}'
+PATCH=$(cat <<EOPATCH
```

Comment:
Quotes `"` are missing around `$()`.

Can we use heredoc instead of cat?

Reply:
Or even better: this patch is static and does not contain any variables - can we save it as a separate json file?

Reply:
Done.